src/main/kotlin/com/modules/constants/AppConsts.kt

```diff
@@ -0,0 +1,50 @@
+package com.modules.constants
+
+object AppConsts {
+    const val SESSION_LENGTH: Long = 60
```

Comment:
```suggestion
    const val SESSION_LENGTH_SECONDS: Long = 60
```

albo można użyć Duration